_posts/2019-06-03-omero-5-5.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: OMERO 5.5 Status Update
+categories: future-plans community
+---
+
+After the intensive development period of IDR’s first releases, the 5.4 series of OMERO was intended to be a stable platform for the community and the OME team to build on. From its first release in October 2017 to its tenth and final release this year, 5.4 has, we think, served as a reference point for the community.
+
+In trying to maintain that stability, however, it’s become ever more clear that we need the ability to quickly release individual parts of OMERO to the community. Fixes to file formats, performance improvements, security patches, and more should not need to wait on the simultaneous release of the entire OMERO platform.
+
+Enabling such releases has been the focus of the upcoming, largely developer-centric release. Though with production-quality docker deployments and the fresh-off-the-presses Bio-Formats 6.1, we hope that OMERO 5.5 will provide something for everyone.
+
+### Individual Repositories
+
+During the development of OMERO 5.5, all 800k+ lines of Java and Matlab code were migrated out of the [openmicroscopy/openmicroscopy](https://github.com/openmicroscopy/openmicroscopy) GitHub repository into individual repositories, each with a new [Gradle](https://gradle.org) build system, and the removal of support for Java 7, Python 2.6, Ice 3.5. Java 11 is now supported. The versions of most of these new repositories began at 5.5.0, but they have already begun to diverge following [semver](http://semver.org) principles. Though initially disruptive, we hope this modernization and modularization will ease participation in the development of OMERO. See the Gradle super project [omero-build](https://github.com/ome/omero-build) for more details.
+
+### Docker Deployments
+
+Beyond the changes for building OMERO, the packaging of OMERO.web and OMERO.server into docker images has now graduated to production quality. Examples for using these images in various configurations are available under [omero-deployment-examples](https://github.com/ome/omero-deployment-examples). Both images are released with every openmicroscopy/openmicroscopy but will also be updated with releases of the embedded components as necessary. Other docker images from the OME team that you may have used over the years have been deprecated and will soon be removed. A next step will be to additionally provide Helm Charts for easing deployment on Kubernetes.
```

Comment:
```suggestion
Beyond the changes for building OMERO, the distribution of [OMERO.server](https://hub.docker.com/r/openmicroscopy/omero-server) and [OMERO.web](https://hub.docker.com/r/openmicroscopy/omero-web-standalone/) as Docker images is now considered production quality. Examples for using these images in various configurations are available under [omero-deployment-examples](https://github.com/ome/omero-deployment-examples). Both images will be updated with every OMERO release, and will also be updated with releases of the embedded components and plugins as necessary.

Other docker images from the OME team that you may have used over the years have been deprecated and will soon be removed. A next step will be to additionally provide Helm Charts for easing deployment on Kubernetes. If you are interested please get in touch through the [forum](https://forum.image.sc/tags/ome).
```

Reply:
We'll need to ensure https://github.com/ome/omero-deployment-examples is updated before this post is released. We've started moving some examples into separate repos.

Reply:
What is the time? since we need the blog post out before 5.5 too